tests/TODO.md

```diff
@@ -0,0 +1,35 @@
+### DESCRIPTION
+
+This file contains a list of test cases and notes on features which should
+be covered with automatic tests.
+
+The list is broken down by sections each of which groups tests related to some
+functional unit or feature. The format of list is as follows:
+
+#### %Name of a feature or a functional unit%:
+
+* **%Test title%**. %Short description: what this test checks%.
+  1. *%Title of test case #1%*. %Brief description of setup, action, and expected
+  result%;
+  2. *%Title of test case #2%*. `...`
+
+### TESTS
+
+#### Couple:
+
+* **DC sharing check**. When option `forbidden_dc_sharing_among_groups` is enabled,
+mastermind checks whether none of pairs of couple's groups has backends in the
+same DC. The check is made in method `Couple::check_dc_sharing` which is
+invoked by `Couple::update_status`.
+  1. *Basic setup*. There are three groups each of which has one backend. All
```

Comment:
What does this test case check, in human words? In facts, it verifies that after some actions some function returns `OK`, but what is the point of this test?